spec/views/articles_spec.rb

```diff
@@ -47,4 +47,11 @@ def create_comment(parent_id = nil)
     expect(rendered).to have_css("form#new_comment")
     expect(rendered).to have_css("input#submit-button")
   end
+
+  it "shows a note about the canonical URL" do
+    allow(article1).to receive(:canonical_url).and_return("https://example.com/lamas")
+    render
+    expect(rendered).to have_text("Originally published at")
```

Comment:
`rendered` is full of newlines and whitespace, so I just match twice. Alternatively we can match on `/\AOriginally published at\s+example.com\z/`, LMK which one you prefer.